Jenkinsfile

```diff
@@ -48,6 +48,11 @@ pipeline {
                 sh "python add_redirects.py $major_version $minor_version stan-users-guide"
             }
         }
+        stage("Link docs to latest") {
+            steps{
+                sh "add_links.sh $major_version $minor_version"
```

Comment:
is this the correct version w/r/t the release, e.g., for 2.23 release, will this add links to 2.22 release?
also, this operation can only be done once, otherwise the HTML will contain multiple versions of the link-to-latest message - so only do this if $patch == 0?

Reply:
I'm now adding a step to extract last version of the docs. We don't rebuild docs on patches ? For example now we'll have 2.22.1 but the docs stay the same so we won't need if $patch == 0.

Reply:
Yes, I don't think we'll need different docs releases for patch releases.

Reply:
correct, not versioning docs for patch releases.  patch releases fix bugs in code.  docs describe what release does - therefore, docs not expected to change.